NukeTestRunner/run_tests.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
```

Comment:
One more adjustment, in python the naming convention for packages is all lowercase. So this would mean it would be called `nuke_test_runner`. My bad if I called it wrong the previous comment